packages/commerce-sdk-react/src/hooks/ShopperBaskets/action.ts

```diff
@@ -353,5 +353,5 @@ export function useShopperBasketsAction<Action extends ShopperBasketsActions>(
     const method = client[action]
     assertMethod(method)
 
-    return useAsyncExecute((arg: Arg) => method.call(client, arg))
+    return useAsyncExecute((...arg: Arg) => method.call(client, arg))
```

Comment:
This change allows users to pass the `rawResponse` flag, which means that the hook would possibly return a `Response` object. Is that something that we want to allow? If so, the type assertion and return type of the hook need to be updated. (The type assertion is why this change didn't cause TypeScript to complain. This is a good example of why they're dangerous!) If we don't want them to change, but you want to keep the multiple argument signature of `useAsyncEffect`, then just change the type passed to `ActionResponse` to a tuple, i.e. `ActionResponse<[Argument<Client[Action]>], ...>`.